src/annotator/util/test/scroll-test.js

```diff
@@ -128,5 +128,37 @@ describe('annotator/util/scroll', () => {
         delete document.body.tagName;
       }
     });
+
+
+    // A test for scrolling when there is a target within the 'details' tag.
+    it('scrolls element into view when the target is within the details tag',async () => {
+      const container = document.createElement('div');
+      container.style.height = '500px';
+      container.style.overflow = 'auto';
+      container.style.position = 'relative';
+      document.body.append(container);
+
+      const details = document.createElement('details');
+      details.style.position = 'absolute';
+      details.style.top = '1000px';
+      container.append(details);
+
+      const summary = document.createElement('summary');
+      summary.append('Summary');
+      details.append(summary);
+
+      const target = document.createElement('div');
+      target.style.height = '20px';
+      target.style.width = '100px';
+      details.append(target);
+
+      await scrollElementIntoView(target, { maxDuration: 1 });
+
+      const containerRect = container.getBoundingClientRect();
```

Comment:
The main code change in `scroll.ts` is to open the `<details>` element. A check for that is missing here though.

Reply:
I'm grateful for catching something so crucial. I've included a test now.